CONTRIBUTORS.md

```diff
@@ -6,35 +6,27 @@ The following people have contributed to Chapel releases:
 Contributors to the Chapel 2.1 release
```

Comment:
Don't forget to update this to the current release number :)

Reply:
When will I learn…?  🤦   Thanks Lydia!

Reply:
I don't think you forget *that* often XD

Reply:
I think the git history would prove you wrong!  :D